integration-cli/docker_cli_daemon_test.go

```diff
@@ -1825,7 +1825,7 @@ func (s *DockerDaemonSuite) TestDaemonNoSpaceLeftOnDeviceError(c *check.C) {
 	defer s.d.Stop(c)
 
 	// pull a repository large enough to fill the mount point
-	pullOut, err := s.d.Cmd("pull", "registry:2")
+	pullOut, err := s.d.Cmd("pull", "debian:stretch")
```

Comment:
registry:2 didn't have a manifest list either, so I switched it to debian:stretch. The entire point of this test though is to run it out of memory so that seems fine to me